dbt/include/bigquery/macros/relations/materialized_view/describe.sql

```diff
@@ -0,0 +1,45 @@
+{% macro bigquery__describe_materialized_view(relation) %}
+    {%- set _materialized_view_sql -%}
+        -- checks each column to see if its a cluster_by field then adds it to a new list
+        with ClusteringColumns as (
+            select
+                table_name,
+                ARRAY_AGG(
+                case
+                    when clustering_ordinal_position is not null then column_name
+                    else null
+                end
+                ignore nulls
+                ) as clustering_fields
+            from
+                `{{ relation.database }}.{{ relation.schema }}.INFORMATION_SCHEMA.COLUMNS`
+            where
+                table_name = '{{ relation.name }}'
+            GROUP BY
+                table_name
+)
+        select
+            mv.table_name as materialized_view,
+            c.column_name,
+            c.is_partitioning_column,
+            c.clustering_ordinal_position,
+            topt.option_name,
+            topt.option_value,
+            topt.option_type
+        from
+            `{{ relation.database }}.{{ relation.schema }}.INFORMATION_SCHEMA.MATERIALIZED_VIEWS` mv
+        left join
+            `{{ relation.database }}.{{ relation.schema }}.INFORMATION_SCHEMA.COLUMNS` c
+        on
+            mv.table_name = c.table_name
+        left join
+            `{{ relation.database }}.{{ relation.schema }}.INFORMATION_SCHEMA.TABLE_OPTIONS` topt
+        on
+            mv.table_name = topt.table_name
+        where
+            mv.table_name = '{{ relation.name }}'
+    {%- endset %}
+    {% set _materialized_view = run_query(_materialized_view_sql) %}
+
+    {% do return({'materialized_view': _materialized_viewy}) %}
+{% endmacro %}
```

Comment:
Working toward getting all options here. 

have been able to produce the `cluster_by` still trying to get a similar cte for `partitions` and find a way to tag the other full list of `options` on materialized views. some seem tied to the `information_schema.table_options` but are for external tables only. also looked at tapping directly into `Tables` or `QueryJobConfig` classes but only found pythonic ways to get that information.

or should I be trying to to use python methods like `get_table_options`, `get_common_options`